coretex/folder_manager.py

```diff
@@ -68,11 +69,15 @@ def _createFolder(name: str) -> Path:
 temp             = _createFolder("temp")
 _artifactsFolder = _createFolder("artifacts")
 
-runsLogDirectory = logs / "runs"
-runsLogDirectory.mkdir(exist_ok = True)
+if not isCliRuntime():
```

Comment:
No need to do this in folder_manager, just remove the place where storagePath gets overriden.